docs-src/content/_index.md

```diff
@@ -2,11 +2,226 @@
 title: "Cubic SDK Documentation"
 ---
 
-# Cubic SDK
+# Cubic API Overview
 
-Cubic is Cobalt's speech recognition engine.  It can be bundled into your
-application as a library, or deployed on-prem and accessed over the network.
-This documentation refers to accessing the network-based Cubic server.
+Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support Go, Python and Java, and are adding support for more languages.
+
+Once running, Cubic’s API provides a method to which you can stream audio. This audio can either be from a microphone or a file. We recommend uncompressed WAV as the encoding, but support other formats such as MP3.
```

Comment:
The current list is MP3, FLAC, WAV, and then single channel raw files (headerless) with the formats of ULAW, VOX, and 16-bit PCM.  I don't know if you want to include all of those, but there you go.  :)